frontend/components/flashcard/Flashcard.tsx

```diff
@@ -33,6 +33,10 @@ interface FlashcardProps {
 	 * Flashcard side to show
 	 */
 	cardSide?: number;
+	/**
+	 * permissions to edit
```

Comment:
```suggestion
	 * Permissions to edit
```
inofficial project standard